src/components/multi-select/multi-select.hbs

```diff
@@ -21,17 +21,18 @@
       </svg>
     </div>
   </div>
-  <div class="{{@root.prefix}}--list-box__menu">
+  <fieldset class="{{@root.prefix}}--list-box__menu">
+    <legend class="{{@root.prefix}}--visually-hidden">Description of form elements within the fieldset</legend>
```

Comment:
Could we use `bx--assistive-text` here? The classes are equivalent but I believe the intended usage may be different.